From aad6fb37dda5dcafb5dbc38fd88c629f148aef1b Mon Sep 17 00:00:00 2001
From: Justin Maggard <jmaggard@netgear.com>
Date: Wed, 18 Nov 2015 12:28:57 -0800
Subject: [PATCH 1014/1016] btrfs: don't automatically resume balance

There may be balance-related mount issues, so don't automatically resume
a running balance.  Just leave it paused.
---
 fs/btrfs/volumes.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/btrfs/volumes.c b/fs/btrfs/volumes.c
index 9420b17..48b1c02 100644
--- a/fs/btrfs/volumes.c
+++ b/fs/btrfs/volumes.c
@@ -3864,7 +3864,7 @@ int btrfs_resume_balance_async(struct btrfs_fs_info *fs_info)
 	}
 	spin_unlock(&fs_info->balance_lock);
 
-	if (btrfs_test_opt(fs_info->tree_root, SKIP_BALANCE)) {
+	if (true) {
 		btrfs_info(fs_info, "force skipping balance");
 		return 0;
 	}
-- 
1.9.1

